CI/include/build_support.sh

```diff
@@ -165,7 +206,12 @@ github_fetch() {
         fi
 
     else
-        git clone "https://github.com/${GH_USER}/${GH_REPO}.git" "$(pwd)"
+        if [ "${CI}" ] && [ "$(git_has_sparse_checkout)" ] && [ "${GIT_OPT_SPARSE}" ]; then
```

Comment:
Here, I think I was trying to avoid a really long line, and I tend to lean towards breaking things into functions if they could conceivably be/become common functionality.